### PR TITLE
Kh cted 1783

### PR DESCRIPTION
**Pull request by kurvinh**

### CODE REVIEW

**Review by BrendonPretorius — APPROVED** [review]

File: amd/src/mod_hsuforum_focus.js [hunk 1]
```diff
@@ -33,6 +33,24 @@
...
+                        $('#' + collapsable[x].id).removeClass('show')
+                        $('.expandalldiscussions').removeClass('expanded')
+                    }
+                } else {
+                    $('.expandalldiscussions')[0].innerHTML = 'Collapse All'
+                    const collapsable = $('.posts-collapse-container.collapse');
+                    for (let x = 0; x < collapsable.length; x++) {
+                        $('#' + collapsable[x].id).addClass('show')
+                        $('.expandalldiscussions').addClass('expanded')
+                    }
```
Comment (JJSwanevelder) [inline amd/src/mod_hsuforum_focus.js:51]: Did you try my suggestion in the implementation plan? Think that does the same with less code.

**Review by JJSwanevelder — CHANGES_REQUESTED** [review]
Clarification question

File: amd/src/mod_hsuforum_focus.js (hunk 1 above)
Comment (kurvinh) [reply amd/src/mod_hsuforum_focus.js]: I used your function, still needs to be run on every toggable element no the page so no real change in code length

**Review by JJSwanevelder — APPROVED** [review]
👍

**Pull request merged**